addons/sales_team/tests/test_sales_team_membership.py

```diff
@@ -303,3 +303,28 @@ def test_users_sale_team_id(self):
         self.sales_team_1_m1.unlink()
         self.assertEqual(self.user_sales_leads.crm_team_ids, self.new_team)
         self.assertEqual(self.user_sales_leads.sale_team_id, self.new_team)
+
+    def test_sales_team_member_search(self):
+        # create a team
```

Comment:
I would argue, for once, that there are too many comments ^^'

I'm partial to only adding comments to explaine the "why" not the "what", unless the "what" is particularly complex. I think you should just add a docstring to explain what the test checks globally, and get rid of the comments. Something like "Check that removed team members do not appear in search results.".

Reply:
updated